README.md

```diff
@@ -214,6 +214,37 @@ For example, when loading and authorizing a `Post` resource which can have one o
 to load and authorize the parent `Post` resource using the `post_id` in /posts/:post_id/comments before you
 create the `Comment` resource using its parent.
 
+### Specifing database field
+You can use `:id_field` option if you want to use other field name instead of the default `:id` for searching the resource. Note that the field specified must be unique among the resource table.
```

Comment:
I think this might be clearer:

You can tell Canary to search for a resource using a field other than the default `:id` by using the `:id_field` option. Note that the specified field must be able to uniquely identify any row in the specified table.